bagisto/bagisto
# Issue Fixed #6822

## Issue Reference

#6822 
## Description

Invoice Table Responsive issue fixed. Added Responsive table class in invoice table to getting Fixed this issue.
## How To Test This?

Step to Test:
1. Go to Customer Profile -> Orders Section -> View Order -> Invoice -> Inspect in mobile view -> Check the table -> Scroll Bar is showing now
## Documentation
